server/src/main/java/org/opensearch/action/admin/cluster/snapshots/status/SnapshotStatus.java

```diff
@@ -86,34 +86,40 @@ public class SnapshotStatus implements ToXContentObject, Writeable {
 
     private SnapshotStats stats;
 
+    private final long initialTotalSizeInBytes;
+
     @Nullable
     private final Boolean includeGlobalState;
 
     SnapshotStatus(StreamInput in) throws IOException {
         snapshot = new Snapshot(in);
         state = State.fromValue(in.readByte());
         shards = Collections.unmodifiableList(in.readList(SnapshotIndexShardStatus::new));
+        initialTotalSizeInBytes = in.readLong();
```

Comment:
Will this be backwards compatible? For a V1 snapshot, this value would not be present, right?